conftest.py

```diff
@@ -435,6 +435,25 @@ def small_client(
             yield client
 
 
+def get_cluster_memory(client: distributed.Client) -> int:
+    "Total memory available on the cluster, in bytes"
+    return int(
+        sum(w["memory_limit"] for w in client.scheduler_info()["workers"].values())
+    )
+
+
+@pytest.fixture(
+    params=[
+        pytest.param(0.3, id="30% cluster memory"),
```

Comment:
30% is just a guess. I'm looking for stuff that "comfortably fits into memory". I figured that if a test actually generates that much data, we could hold two full copies before starting to spill.

This will be individual for every test though since most tests still multiply the cluster-memory by a certain factor etc.